org.lflang/src/lib/ts/package.json

```diff
@@ -2,20 +2,15 @@
     "name": "LinguaFrancaDefault",
     "version": "0.0.1",
     "description": "A default Lingua Franca project for the TypeScript target",
+    "type": "commonjs",
```

Comment:
I think this was resolved in Zulip. TypeScript can transpile to either ES Modules or CommonJS, and we should transpile to CommonJS because CommonJS is older and compatible with more things.

Reply:
OK, sounds good to me.